cmd/forwarder/fixer.go

```diff
@@ -111,9 +110,12 @@ func fix(req *http.Request, r io.Reader, remoteAddr string, logplexDrainToken st
 		messageWriter.WriteString(" ")
 		//messageWriter.Write(header.Msgid)
 		_ = writeField(&messageWriter, header.Msgid, 32)
-		messageWriter.WriteString(" [origin ip=\"")
-		messageWriter.WriteString(remoteAddr)
-		messageWriter.WriteString("\"]")
+		messageWriter.WriteString(" ")
```

Comment:
btw @mmowris we can improve the efficiency here by merging this line and the one above it into a single line, ie:

```go
  messageWriter.WriteString("\"] ")
```